newsroom/templates/footer.html

```diff
@@ -0,0 +1,6 @@
+<footer class="footer">
+    <a href="https://www.aap.com.au/legal/" target="_blank">{{ gettext('Privacy Policy') }}</a>
+    <a href="https://www.aap.com.au/legal/" target="_blank">{{ gettext('Terms and Conditions') }}</a>
+    <a href="https://www.aap.com.au/contact/sales-inquiries/" target="_blank">{{ gettext('Contact Us') }}</a>
+    <a href="/copyright">{{ gettext('Copyright') }}</a>
```

Comment:
would be good to use `{{ url_for('public.copyright') }}` so we know where it's defined